sccm/core/servers/manage/cmpivot.md

```diff
@@ -224,6 +224,20 @@ The queries and the results are all just text. The entities **InstallSoftware**
 
 A query times out after one hour. For example, a collection has 500 devices, and 450 of the clients are currently online. Those active devices receive the query and return the results almost immediately. If you leave the CMPivot window open, as the other 50 clients come online, they also receive the query, and return results. 
 
+>[!TIP]
+> CMPivot interations are logged to the following log files:
+
+> Server Side:
+> - SmsProv.log
+> - bgbServer.log
+> - StateSys.log
+
+> Client Side:
+> - CcmNotificationAgent.log
+> - Scripts.log
+> - StateMessage.log
+
+> [For more Information on Configuration Manger log files](/sccm/core/plan-design/hierarchy/log-files)
```

Comment:
Please revise this to: `For more information, see [Log files](/sccm/core/plan-design/hierarchy/log-files).`